security/identity-and-access-management/oracle-identity-governance/postman-scim-samples/files/OIG_POSTMAN_SAMPLES.postman_collection.json

```diff
@@ -0,0 +1,305 @@
+{
+	"info": {
+		"_postman_id": "2b5ecb24-88a5-4390-8246-0b848f6c6050",
```

Comment:
Not entirely sure and I wouldn't modify it, as it might be expected/enforced in future or past Postman versions. Note that this JSON and its structure is generated on export by Postman, so changing anything by hand in the file is highly unrecommended.